yivi-docs/blog/2025-07-01-post-mortem/index.md

```diff
@@ -0,0 +1,75 @@
+| June 30th, 2025, 15:30 CEST       | Changed the `irma.app` DNS settings back to the old server allowing Apple devices to check the site association again.                                                                                                          |
+| June 30th, 2025, 16:00 CEST       | Confirmed that the issue was resolved; the Yivi app was able to open on iOS devices again.                                                                                                           |
+| July 1st, 2025, 11:45 CEST        | Delivered a workaround to affected users by providing a page that converts `irma.app` Universal Links to `open.yivi.app` Universal Links.                                                           |
+| July 1st, 2025, 14:00 CEST        | Documented the issue and resolution in a post-mortem on our blog and shared it with the community.                                                                                                   |
+
+
+## Customer impact
+The incident affected a small number of iOS users who were unable to open the Yivi app due to the Universal Links issue. The impact was limited to users who installed the app after the domain migration, as existing installations were not affected. According to our `keyshare server registrations`, the number of affected users was about `78 users` including Android users, which is a small percentage of our total user base.
+
+## Root cause and mitigation
+As part of our efforts to improve the Yivi app, we migrated the domain `irma.app` to `yivi.app`. This migration was intended to provide a more consistent branding experience for our users and to gain control over all domains originally maintained by `SIDN`. However, the migration inadvertently caused an issue with the Universal Links feature on iOS devices, which do not support forwarding for Universal Links.
+
+The Universal Links feature allows the Yivi app to open directly from links in the browser, providing a seamless user experience. However, iOS requires that the associated domain file (`apple-app-site-association`) be hosted on the domain without any redirects. The redirection of traffic from `irma.app` to `yivi.app` caused the associated domain file to be redirected, which prevented the Yivi app from opening on iOS devices that installed the app after the migration.
+
+> `https://fully qualified domain/.well-known/apple-app-site-association`
+> You must host the file using https:// with a valid certificate and with no redirects.
+
+[Link to Apple's documentation](https://developer.apple.com/documentation/xcode/supporting-associated-domains#:~:text=and%20with%20no-,redirects,-.)
+
+The reason it was not breaking Android devices is that Android does not require the associated domain file to be hosted without redirects. Android uses a different mechanism for handling Universal Links, which allows it to work even if the domain is redirected.
```

Comment:
[According to the documentation](https://developer.android.com/training/app-links/verify-android-applinks), Android doesn't allow redirect for `assetslinks.json` as well.